shared/menubar/index.desktop.js

```diff
@@ -55,7 +55,7 @@ class MenubarRender extends Component<Props, State> {
     const styles = stylesPublic
 
     const menuColor = this.state.showingMenu ? globalColors.black_60 : globalColors.black_40
-    const menuStyle = {...desktopStyles.clickable, color: menuColor, hoverColor: menuColor}
+    const menuStyle = collapseStyles([desktopStyles.clickable])
```

Comment:
no need to collapse just one, I think you want `platformStyles` though since this is desktop-only

Reply:
Flow won't allow just assigning this style without using one of these functions IIRC